### ISSUE TITLE
Gradlewrapper jar missing in git repo

### ISSUE THREAD

**Issue by Pieterv24** [pahimar/Equivalent-Exchange#57]
### What should have happened
gradlew should have run
### What actually happened
'Error: Could not find or load main class org.gradle.wrapper.GradleWrapperMain'
was thrown
### What steps do we have to follow so we can reproduce the issue
clone a new version of the git repo and try to setup a dev environment.
### Version of the mod
...
### Pastebin of relevant log files (fml-latest-client.log, fml-latest-server.log, latest.log)
....

Sorry if this was intended behaviour.
Just cross referenced a bit with the EE3 repo.

**Comment by jamierocks** [comment]
Yep. Looks as if https://github.com/pahimar/Equivalent-Exchange/blob/develop/1.12/.gitignore#L4 is a little too strict.
